Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.UIElements.cs

```diff
@@ -251,6 +256,7 @@ public void BindImage(string name, IProperty<Texture> prop)
             Bind<Image, Texture>(name, prop, SetImage);
         }
 
+#if BINDINGS_WEB_REQUEST && BINDINGS_WEB_REQUEST_TEXTURE
```

Comment:
I think (but am not sure) that if you add the package you need as a dependency then Unity will add it to the packages-lock if it's missing. I think that makes sense for this package.

If not, maybe consider having a LogError that recommends adding the package instead of removing the method altogether